win32ss/user/ntuser/draw.c

```diff
@@ -1098,7 +1102,13 @@ DrawFrameControl(HDC hDC, LPRECT rc, UINT uType, UINT uState)
         case DFC_CAPTION:
             return UITOOLS95_DrawFrameCaption(hDC, rc, uState);
         case DFC_MENU:
-            return UITOOLS95_DrawFrameMenu(hDC, rc, uState);
+            FillRect(hDC, rc, (HBRUSH)NtGdiGetStockObject(WHITE_BRUSH)); /* Fill by white */
+            rgbOldText = IntGdiSetTextColor(hDC, RGB(0, 0, 0)); /* Draw by black */
```

Comment:
Are you sure this has to always be black? If you change the color scheme of Windows/ReactOS, where the menu text colors are not black, would this still be drawn black?

Reply:
These drawing functionality of `DFC_MENU` is provided for monochrome image. We use `UITOOLS95_DrawFrameMenu` in drawing menu.

Reply:
![2](https://user-images.githubusercontent.com/2107452/196039190-1eeb8ee4-bbb6-4afd-809c-01d5e6fb7657.png)
![1](https://user-images.githubusercontent.com/2107452/196039192-5d7c8f59-8084-48d2-8683-f8366ecb5688.png)